pilot/control/payloads/generic.py

```diff
@@ -28,6 +29,40 @@ def __init__(self, args, job, out, err):
         self.__out = out
         self.__err = err
 
+    def get_job(self):
+        """
+        Get the job object.
+
+        :return: job object.
+        """
+        return self.__job
+
+    def get_corecount(self):
```

Comment:
Same comment about the necessity of `get_corecount` wrapper, why not to query `infosys.queuedata.corecount` directly from related code?

Reply:
ok. I will update it.

Reply:
another point: do we need to keep initial `zip_time_gap`, I mean does it considered to be used apart from `es_stageout_gap`. If the `zip_time_gap` is only related to ES and not supposed to be used directly, then we could remove it from queuedata (as an attribute) and use it only as an external name in the map to populate `es_stageout_gap`,
what do u think?

Reply:
es_stageout_gap is just an alias of zip_time_gap.

I added one line here.
https://github.com/PanDAWMS/pilot2/blob/next/pilot/info/queuedata.py#L99

Reply:
for corecount, locally I already moved to use infosys. Will be pulled out together with other updates after some tests.

Reply:
for what concern the `es_stageout_gap`: ok clear, 
in this case to avoid confusions please remove `zip_time_gap` from the fields:
here 
https://github.com/PanDAWMS/pilot2/blob/next/pilot/info/queuedata.py#L58
and here:
https://github.com/PanDAWMS/pilot2/blob/next/pilot/info/queuedata.py#L62